code/datums/craft/recipes/weapon.dm

```diff
@@ -186,14 +186,30 @@
 	result = /obj/item/melee/energy/sword/sabre/dagger
 	icon_state = "gun"
 	steps = list(
-		list(/obj/item/tool_upgrade/productivity/ergonomic_grip, 1, "time" = 5),
-		list(/obj/item/stock_parts/subspace/crystal, 1),
-		list(/obj/item/gun/energy/gun/martin, 1, "time" = 5),
+		list(/obj/item/trash/material/e_sword_cutlass, 1, "time" = 5), // Repurposing!
+		list(CRAFT_MATERIAL, 1, MATERIAL_STEEL),
+		list(QUALITY_SAWING, 15, 10), // Reduce the size
+		list(CRAFT_MATERIAL, 3, MATERIAL_PLASTIC),
+		list(QUALITY_HAMMERING, 10, 10), // Adjust the new shape
 		list(/obj/item/cell/small/moebius/nuclear, 1),
 		list(/obj/item/stack/cable_coil, 5, "time" = 5),
 		list(QUALITY_ADHESIVE, 15, 70)
 	)
 
+/datum/craft_recipe/weapon/energy_spear
+	name = "energy spear"
+	result = /obj/item/melee/energy/spear
+	icon_state = "gun"
+	steps = list(
+		list(/obj/item/stack/rods, 1, "time" = 15),
+		list(/obj/item/trash/material/e_sword_cutlass, 1, "time" = 5), // Let's tape it to a rod and call it a spear!
```

Comment:
just checking if you meant the spear to use a trash esword base

Reply:
Nope, the "base" is a rod, then you line up the trashed up esword to its tip and tape it, then repair it, the rod is meant to be the base, not the trash sword!